xml/System.Numerics.Tensors/TensorPrimitives.xml

```diff
@@ -3405,11 +4018,14 @@
       </Parameters>
       <Docs>
         <typeparam name="T">To be added.</typeparam>
-        <param name="x">To be added.</param>
-        <param name="y">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="x">The first tensor, represented as a span.</param>
+        <param name="y">The second tensor, represented as a span.</param>
+        <summary>Computes the bitwise Hamming distance between two equal-length tensors of values.</summary>
+        <returns>The number of bits that differ between the two spans.</returns>
         <remarks>To be added.</remarks>
+        <exception cref="T:System.ArgumentException">Length of <paramref name="x" /> must be same as length of <paramref name="y" />.</exception>
```

Comment:
These should be combined and reworded as "Length of x is not the same as length of y" and "x or y is empty".